zold_wallet/pubspec.yaml

```diff
@@ -32,4 +32,10 @@ flutter:
 flutter_icons:
   android: "launcher_icon" 
   ios: true
-  image_path: "assets/icon/icon.png"
\ No newline at end of file
+  image_path: "assets/icon/icon.png"
+
+  fonts:
+  - family: Monospace
+    fonts:
+      - asset: assets/fonts/Monospace.ttf
```

Comment:
@amorenew you should add the `Monospace.ttf` file to the assets, right?

Reply:
I fixed it and used Roboto Mono from google material
I found that Monospace is a general name for fonts style and matched monospaced font is Roboto Mono in material design fonts